src/preprocessing/passes/ackermann.cpp

```diff
@@ -174,10 +175,13 @@ void collectFunctionsAndLemmas(FunctionToArgsMap& fun_to_args,
                                   nm,
                                   vec);
       }
+      else if (term.getKind() == Kind::STORE)
+      {
+        throw OptionException("Ackermannization is not supported for kind: "
```

Comment:
I believe this should be a LogicException.
As I have understood in the past, "OptionException" is that a bad combination of options was used, independent of the content of the problem.

Reply:
Sure, fixed.